types/crypto-random-string/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
In new type definition, use only extends from dtslint/dt.json and remove everything else please.

Reply:
Done. Should be in line with the guidelines now. Sorry for the inconvenience.